src/tests/base_test.py

```diff
@@ -123,7 +166,7 @@ def setUp(self):
             email="test.user.societies@andela.com",
             country=self.nigeria,
             cohort=self.cohort_1_Nig,
-            society = self.phoenix)
+            society=self.phoenix)
 
         self.test_user_2 = User(
             uuid="-KdQsawesome_useridZ",
```

Comment:
Get rid of `role="president"` at line 187 to fix the error. Also use the newer `initial_data.py` you shared on DM and make the minor changes.